atom/browser/api/atom_api_window.cc

```diff
@@ -83,7 +83,7 @@ Window::Window(v8::Isolate* isolate, v8::Local<v8::Object> wrapper,
   v8::Local<v8::Value> transparent;
   if (options.Get("transparent", &transparent))
     web_preferences.Set("transparent", transparent);
-
```

Comment:
You can (and probably should) have the line break.

But you had some stray " " characters on that line.  And it's against the style guide to have space characters at the end of a line.  Let alone on an empty line 👍